README.md

```diff
@@ -8,28 +8,28 @@ https://stashapp.cc
 
 **Stash is a locally hosted web-based app written in Go which organizes and serves your porn.**
 
-* It can gather information about videos in your collection from the internet, and is extensible through the use of community-built plugins.
-* It supports a wide variety of both video and image formats
+* It can gather information about videos in your collection from the internet, and is extensible through the use of community-built plugins for a large number of content producers.
+* It supports a wide variety of both video and image formats.
 * You can tag videos and find them later.
 * It provides statistics about performers, tags, studios and other things.
 
 You can [watch a demo video](https://vimeo.com/275537038) to see it in action (password is stashapp).
 
-For further information you can [read the in-app manual](https://github.com/stashapp/stash/tree/develop/ui/v2.5/src/docs/en).
+For further information you can [read the in-app manual](tree/develop/ui/v2.5/src/docs/en).
```

Comment:
This gives a 404 error when I try to navigate to it. If we're using relative paths, we may as well use the existing branch. It should probably be `ui/v2.5/src/docs/en`.